lib/email_alerts.py

```diff
@@ -32,6 +35,8 @@ def SplitEmailsAndAppendEmailDomain(address_list):
   """Splits a string of comma-separated emails, appending default domain."""
   result = []
   # Process email addresses, and build up a list.
+  if isinstance(address_list, rdfvalue.DomainEmailAddress):
+    address_list = str(address_list)
```

Comment:
It worked because it was a string after this so the next condition turned it into an array, but changed to be more obvious.